etn-ccis/blui-react-workflows
# Feature/4448 maintain app state

Fixes 4448 .



#### Changes proposed in this Pull Request:

- Click on debug button of login screen and change app language 
- log in into app and go to homepage same language should be throughout app
- maintain state of app after refreshing using localstorage



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

-Click on debug button of login screen and change app language 
- log in into app and go to homepage same language should be throughout app
- refresh homepage it should be on homepage only
- 



#### Any specific feedback you are looking for?

- this needs to be merge https://github.com/etn-ccis/blui-react-workflows/pull/366 first
